### PR TITLE
AV-214390 passthrough objects should not create GS in customFQDN mode

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Review by DixitAakash — COMMENTED** [review]
Have minor comments, otherwise PR looks good.

File: gslb/k8sobjects/route_object.go [hunk 1]
```diff
@@ -284,6 +284,11 @@ func (route RouteMeta) ApplyFilter() bool {
 	selectedByGDP := route.ApplyGDPSelector()
 	if selectedByGDP {
 		if gslbutils.GetCustomFqdnMode() {
+			if route.IsPassthrough() {
+				gslbutils.Debugf("cluster: %s, ns: %s, route host: %s, msg: passthrough route not supported in customfqdn mode",
```
[the remark this answers is not in the document]
Comment (arihantg) [reply gslb/k8sobjects/route_object.go]: Updated to Info

File: gslb/k8sobjects/ingress_object.go [hunk 2]
```diff
@@ -331,6 +340,11 @@ func (ihm IngressHostMeta) ApplyFilter() bool {
 	selectedByGDP := ihm.ApplyGDPSelector()
 	if selectedByGDP {
 		if gslbutils.GetCustomFqdnMode() {
+			if ihm.IsPassthrough() {
+				gslbutils.Debugf("cluster: %s, ns: %s, ingress host: %s, msg: passthrough ingress not supported in customfqdn mode",
```
[the remark this answers is not in the document]
Comment (arihantg) [reply gslb/k8sobjects/ingress_object.go]: Updated to Info

**Review by DixitAakash — APPROVED** [review]
LGTM

**Pull request merged**